doc/source/user_guide/index.rst

```diff
@@ -128,4 +131,4 @@ these main use cases for customizing the picker and hover callbacks:
   override the ``hover_select_object`` and ``hover_unselect_object`` methods. These methods are called when an
   object is hovered over or unhovered, respectively.
 
-A practical example of how to use the ``AbstractPicker`` class are included in the examples section of the documentation.
\ No newline at end of file
+A practical example of how to use the ``AbstractPicker`` class are included in the examples section of the documentation.
```

Comment:
I'd suggest linking to the specific example with a sentence link this:

To see how to use the ``AbstractPicker`` class, see the "X" example.